biojava-structure/src/main/java/org/biojava/nbio/structure/secstruc/SecStrucCalc.java

```diff
@@ -683,7 +683,7 @@ public String printFASTA() {
 	public int hashCode() {
 	    final int prime = 31;
 	    int result = 1;
-	    result = prime * result + Arrays.hashCode(atoms);
+	    result = prime * result + Arrays.hashCode(groups);
```

Comment:
I see both of them deal with getGroupTable() methods
What do you recommend it to be ?